syntaxes/graphql.connectors.json

```diff
@@ -0,0 +1,108 @@
+        { "include": "source.graphql#literal-quasi-embedded" },
+        { "include": "source.graphql#graphql-skip-newlines" }
+      ]
+    },
+    "connect-arguments": {
+      "name": "meta.arguments.connect.graphql",
+      "begin": "\\s*(\\()",
+      "end": "\\s*(\\))",
+      "beginCaptures": {
+        "1": { "name": "meta.brace.round.directive.graphql" }
+      },
+      "endCaptures": {
+        "1": { "name": "meta.brace.round.directive.graphql" }
+      },
+      "patterns": [
+        { "include": "source.graphql#graphql-comment" },
+        { "include": "#connect-jsonselection-argument" },
+        { "include": "#connect-string-jsonselection-argument" },
+        {
+          "begin": "\\s*(http)(?:\\s*(:))",
+          "end": "(?=\\s*(?:(?:([_A-Za-z][_0-9A-Za-z]*)\\s*(:))|\\)))|\\s*(,)",
+          "beginCaptures": {
+            "1": { "name": "variable.parameter.graphql" },
+            "2": { "name": "punctuation.colon.graphql" }
+          },
+          "endCaptures": {
+            "3": { "name": "punctuation.comma.graphql" }
+          },
+          "patterns": [
+            { "include": "#connect-arguments" },
+            { "include": "source.graphql#graphql-comment" },
+            { "include": "source.graphql#literal-quasi-embedded" },
+            { "include": "source.graphql#graphql-skip-newlines" }
+          ]
+        },
+        { "include": "source.graphql#graphql-argument" },
+        { "include": "source.graphql#literal-quasi-embedded" }
+      ]
+    },
+    "connect-jsonselection-argument": {
+      "name": "meta.argument.connect.jsonselection.graphql",
+      "begin": "\\s*(selection|body)(?:\\s*(:))",
+      "end": "(?=\\s*(?:(?:([_A-Za-z][_0-9A-Za-z]*)\\s*(:))|\\)))|\\s*(,)",
+      "beginCaptures": {
+        "1": { "name": "variable.parameter.graphql" },
+        "2": { "name": "punctuation.colon.graphql" }
+      },
+      "endCaptures": {
+        "3": { "name": "punctuation.comma.graphql" }
+      },
+      "patterns": [
+        {
+          "contentName": "meta.embedded.block.jsonselection.graphql",
+          "begin": "((\"\"\"))",
+          "end": "((\"\"\"))",
+          "beginCaptures": {
+            "1": { "name": "string.quoted.double.graphql" },
+            "2": { "name": "punctuation.definition.string.begin.graphql" }
+          },
+          "endCaptures": {
+            "1": { "name": "string.quoted.double.graphql" },
+            "2": { "name": "punctuation.definition.string.end.graphql" }
+          },
+          "patterns": [
+            {
+              "include": "source.apollo.jsonselection"
+            }
+          ]
+        },
+        { "include": "source.graphql#graphql-skip-newlines" }
+      ]
+    },
+    "connect-string-jsonselection-argument": {
+      "name": "meta.argument.connect.string.jsonselection.graphql",
+      "contentName": "meta.embedded.block.string.jsonselection.graphql",
+      "begin": "\\s*(GET|POST)(?:\\s*(:))",
```

Comment:
Oh I see, is this what is checking the sub-argument to find `body`? We also support `PUT`, `PATCH`, and `DELETE`

Reply:
This is to find usages where we don't do a full jsonselection "selection set", but a string that contains selections between `{}`.

Reply:
Also, added support for PUT|PATCH|DELETE ✅